tests/unit/email/test_init.py

```diff
@@ -67,7 +72,10 @@ def retry(exc):
                 raise celery.exceptions.Retry
 
         sender, task = FakeMailSender(), Task()
-        request = pretend.stub(find_service=lambda *a, **kw: sender)
+        request = pretend.stub(
+            find_service=lambda *a, **kw: sender,
+            registry=pretend.stub(settings={"site.name": "DevPyPI"}),
```

Comment:
doubt: do we need this here?

Reply:
We do, the subject is still going to be constructed before celery fails.